src/main/java/seedu/address/storage/JsonSerializableRoomList.java

```diff
@@ -42,7 +42,7 @@ public JsonSerializableRoomList(RoomList source) {
      */
     public RoomList toModelType() throws IllegalValueException {
         RoomList roomList = new RoomList();
-        roomList.getRoomDisplayList().clear();
+        //roomList.getRoomDisplayList().clear();
```

Comment:
can remove instead of commenting if this is not going to be used in the future

Reply:
Okay